src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Combination/paginated_list.html.twig

```diff
@@ -35,11 +35,19 @@
     ></div>
 
     <div id="combination-edit-modal"
+         data-empty-image="{{ asset('themes/new-theme/img/empty_state/credit_slip.png') }}"
```

Comment:
Are you missing the correct image here and used this one as a replacement?

Reply:
No, this one is the image inside the mockup :)